nilearn/glm/_base.py

```diff
@@ -83,76 +93,79 @@ def generate_report(
-            & second_level_contrast for SecondLevelModel
-            (:func:`nilearn.glm.second_level.SecondLevelModel.compute_contrast`)
+            Contrasts are passed to ``contrast_def`` for
+            :class:`~nilearn.glm.first_level.FirstLevelModel` through
+            :meth:`~nilearn.glm.first_level.FirstLevelModel.compute_contrast`,
+            and ``second_level_contrast`` for
+            :class:`~nilearn.glm.second_level.SecondLevelModel` through
+            :meth:`~nilearn.glm.second_level.SecondLevelModel.compute_contrast`.
 
-        title : String, optional
-            If string, represents the web page's title and primary heading,
-            model type is sub-heading.
-            If None, page titles and headings are autogenerated
-            using contrast names.
+        title : :obj:`str`, optional
+            - If a :obj:`str`, it represents the web page's title and primary
+              heading, model type is sub-heading.
+            - If ``None``, page titles and headings are autogenerated using
+              contrast names.
 
         bg_img : Niimg-like object, optional
             Default is the MNI152 template (Default='MNI152TEMPLATE')
             See http://nilearn.github.io/manipulating_images/input_output.html
             The background image for mask and stat maps to be plotted on upon.
             To turn off background image, just pass "bg_img=None".
 
-        threshold : float, optional
-            Cluster forming threshold in same scale as `stat_img` (either a
-            t-scale or z-scale value). Used only if height_control is None.
-            Default=3.09
+        threshold : :obj:`float`, optional
+            Cluster forming threshold in same scale as ``stat_img`` (either a
+            t-scale or z-scale value). Used only if ``height_control`` is
+            ``None``. Default=3.09
 
-        alpha : float, optional
+        alpha : :obj:`float`, optional
             Number controlling the thresholding (either a p-value or q-value).
-            Its actual meaning depends on the height_control parameter.
+            Its actual meaning depends on the ``height_control`` parameter.
             This function translates alpha to a z-scale threshold.
             Default=0.001
 
-        cluster_threshold : int, optional
-            Cluster size threshold, in voxels.
+        cluster_threshold : :obj:`int`, optional
+            Cluster size threshold, in :term:`voxels<voxel>`.
             Default=0
 
-        height_control : string or None, optional
-            false positive control meaning of cluster forming
-            threshold: 'fpr' (default) or 'fdr' or 'bonferroni' or None
-            Default='fpr'.
+        height_control : :obj:`str` or None, optional
+            :term:`False positive control<FPR correction>` meaning of cluster
+            forming threshold: 'fpr' (default) or 'fdr' or 'bonferroni' or
```

Comment:
```suggestion
            forming threshold: 'fpr', 'fdr', 'bonferroni' or
```
Remove redundant default statement